tests/conftest.py

```diff
@@ -1,5 +1,11 @@
 import pytest
 from app.create_app import create_app
+from tests.test_application.application_data import expected_application_data
+from tests.test_application.application_data import (
+    expected_application_response,
```

Comment:
is this splitting on 3 lines a required convention? Because it looks ugly to me

Reply:
I run the pre-commit  file with flake 8 & black which split all imports into separate lines 😄

Reply:
thats heart breaking but fine if thats the case 👍🏻

Reply:
Thank you 😄